### PR TITLE
V04 errors intro

### PR DESCRIPTION
**Pull request by michael-scharf**
Updated to the introduction text, based on the branch v04-errors (see PR #21).

### CODE REVIEW

File: draft/draft-scharf-tcpm-yang-tcp.xml [hunk 1]
```diff
@@ -231,190 +224,49 @@
...
+        <t>This document models fundamental system parameters that are
+        configurable on many TCP implementations, and for which the
+        configuration is reasonably similar. Similar to the
+        <xref target="RFC4022">TCP MIB</xref>, this document
+        also specifies a TCP connection table. This enables both
+        global and connection-specific TCP configuration.</t>
+
+        <t>An important use case is the TCP configuration on network elements
+        such as routers, which often use YANG data models. The model therefore
+        specifies TCP parameters that are important  on such TCP stacks. A
```
Comment (mjethanandani) [inline draft/draft-scharf-tcpm-yang-tcp.xml]: An extra blank space on this line.

File: src/yang/ietf-tcp.yang [hunk 2]
```diff
@@ -209,80 +209,89 @@ module ietf-tcp {
...
+    description
+      "Important global TCP stack parameters.";
     leaf mss-max {
       type uint16;
       description
         "Sets the max segment size for TCP connections.";
-      reference "RFC 1122: Requirements for Internet Hosts -- Communication
-        Layers";
+      reference
+         "RFC 1122: Requirements for Internet Hosts -- Communication
```
Comment (mjethanandani) [inline src/yang/ietf-tcp.yang]: The indentation here is off by one. My bad. Could you correct it and the next line.

File: draft/draft-scharf-tcpm-yang-tcp.xml [hunk 3]
```diff
@@ -112,47 +112,24 @@
       augmented to address more advanced or implementation-specific TCP
       features.</t>
 
-      <t>Many protocol stacks on internet hosts use other methods to configure
+      <t>Many protocol stacks on Internet hosts use other methods to configure
       TCP, such as operating system configuration or policies. Many TCP/IP
       stacks cannot be configured by network management protocols such as
       <xref target="RFC6241">NETCONF</xref> or <xref
       target="RFC8040">RESTCONF</xref> and they do not use YANG data models.
```
[the remark this answers is not in the document]
Comment (michael-scharf) [reply draft/draft-scharf-tcpm-yang-tcp.xml]: Fixed in PR #25 

I submitted a new PR as I could not update this branch

File: draft/draft-scharf-tcpm-yang-tcp.xml (hunk 1 above)
Comment (michael-scharf) [reply draft/draft-scharf-tcpm-yang-tcp.xml]: Fixed in PR #25

File: src/yang/ietf-tcp.yang (hunk 2 above)
Comment (michael-scharf) [reply src/yang/ietf-tcp.yang]: Fixed in PR #25

**Comment by michael-scharf** [comment]
This PR is outdated, see PR #25

**Pull request merged**